app/models/foreman_tasks/triggering.rb

```diff
@@ -104,6 +104,7 @@ def parse_start_before!
     private
 
     def can_start_recurring
+      parse_start_at!
```

Comment:
Could we add a validation message instead of just raising?

Reply:
I do not think [parse_start_at!](https://github.com/theforeman/foreman-tasks/blob/master/app/models/foreman_tasks/triggering.rb#L96) raises:

```ruby
Time.zone.parse('foo') #=> nil
```

Reply:
It does ocasionally

```
> Time.zone.parse('2021-08-32')
ArgumentError: argument out of range
from /home/aruzicka/.rvm/gems/ruby-2.7.0/gems/activesupport-6.0.3.7/lib/active_support/values/time_zone.rb:548:in `initialize'

> Time.zone.parse('127.0.0.1')
ArgumentError: mon out of range
from /home/aruzicka/.rvm/gems/ruby-2.7.0/gems/activesupport-6.0.3.7/lib/active_support/values/time_zone.rb:548:in `initialize'
```

Reply:
Updated.